eldritch/events.py

```diff
@@ -2006,22 +2058,41 @@ def BinaryChoice(
 
 class ItemChoice(ChoiceEvent):
 
-  def __init__(self, character, prompt):
+  def __init__(self, character, prompt, decks=None, item_type=items.Item):
     self.character = character
     self._prompt = prompt
     self.choices = None
+    self.chosen = None
+    if decks is None:
+      decks = {'spells', 'common', 'unique', 'skills', 'allies'}
+    assert decks.issubset({'spells', 'common', 'unique', 'skills', 'allies'}), f"{decks} contains invalid deck"
+    # can one choose a skill in any circumstances?
```

Comment:
I'm sure there is. At a minimum, one can choose an ally, for those nasty rumors that ask you to discard an ally.